### PR TITLE
python-script-to-evaluate-gemini

### PR DESCRIPTION
**Pull request by jsaluja**
- Created gemini.py script 
- Refreshed requirements
- Updated .gitignore to exclude streamlit, DS_Store, default.sqlite

### CODE REVIEW

**Comment by jsaluja** [comment]
image
image

File: requirements.txt
Comment (Josephrp) [inline requirements.txt:1]: please here it's important to keep some things together

**Review by Josephrp — CHANGES_REQUESTED** [review]
please keep requirements.txt separate from the trulens requirements;txt

File: requirements.txt
Comment (jsaluja) [reply requirements.txt:1]: fixed

**Comment by Josephrp** [comment]
let's perhaps split requirements.txt into eval and src requirements, so make a new requirements.txt in the evaluation folder ?

**Review by Josephrp — CHANGES_REQUESTED** [review]
let's split the requirements for evaluation to evaluation/requirements.txt and the requirements for src, stays the same, hopefully that's the easiest way to resolve this

**Comment by jsaluja** [comment]
Refactored into https://github.com/Tonic-AI/DataTonic/pull/65

**Pull request closed without merging**